app/lib/test_applications.rb

```diff
@@ -215,18 +218,22 @@ def create_application_to_courses(
       @application_form.update!(work_history_completed: true)
       fast_forward
 
-      # The first reference is declined by the referee
-      @application_form.application_references.feedback_requested.first.update!(feedback_status: 'feedback_refused')
+      if incomplete_references
+        @application_form.application_references.update_all(feedback_status: 'not_requested_yet')
+      else
+        # The first reference is declined by the referee
+        @application_form.application_references.feedback_requested.first.update!(feedback_status: 'feedback_refused')
 
-      # Cancel 1 reference manually and receive feedback on the remaining 2.
-      # Select the two references with feedback.
-      @application_form.application_references.feedback_requested.first.cancelled!
-      @application_form.application_references.feedback_requested.each do |reference|
-        submit_reference!(reference.reload)
-        reference.update(selected: true)
-        fast_forward
+        # Cancel 1 reference manually and receive feedback on the remaining 2.
+        # Select the two references with feedback.
+        @application_form.application_references.feedback_requested.first.cancelled!
+        @application_form.application_references.feedback_requested.each do |reference|
+          submit_reference!(reference.reload)
+          reference.update(selected: true)
```

Comment:
I've just been testing this on Postman. I can see it correctly generates the test data but when you GET an application_choice it only pulls through 'Selected' references'?

Reply:
thanks @JR-G. have rather hackily resolved this by setting incomplete references as `selected = true`. given it's just a temporary thing for testing purposes, i think probably is ok.